### DISCUSSION TITLE
Sync with cloud drive

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [mRemoteNG/mRemoteNG#2448] [Ideas]
I want to use mRemoteNG to store my config on OneDrive which is shared between different machines so that I can have seamless access to a single source of servers, but it seems like each time I start mRemoteNG the connection data is not synced, and so rather it is cached locally. This is confusing so I can't share it to my colleagues because they will get lost easily. I wonder if SQL config store solves this but that is still in beta.

**Comment by Kvarkas** [comment]
curently its a work i am busy with to make a profiles (first step prepare app structure for such) expecting to show in 1.77.4 - that will allow to have a user profile, what will be able to include app configuration bits as well as connections collection (many connection profiles) so hope we could address that in near feature ;)

**Comment by feelnrg** [comment]
I am using FreeFileSync to solve this problem. You can use "external Program" in mRemote to start sync Process for FreeFileSync command. Actually you have to Sync extApps.xml and confCons.xml to work everywhere. Please consider to encrpyt the conCons.xml file :-). It was perfect working for Portable installation. But mRemote was changing the working folders so the confCons.xml is not saved in Portable USB Drive but in Users ../AppData/Roaming folder #bad. Hope this will change in future.

**Comment by Kvarkas** [comment]
yes part of the changes will make new build of app more portable like so its will keep all files in designated places and not try to put them all arround windows system :)
